app/components/agenda/agenda-header/agenda-tabs.js

```diff
@@ -11,27 +10,12 @@ export default class AgendaAgendaHeaderAgendaTabsComponent extends Component {
   @service store;
   @service currentSession;
 
-  @lastValue('loadFirstAgendaitem') firstAgendaitem;
-
-  constructor() {
-    super(...arguments);
-    this.loadFirstAgendaitem.perform();
-  }
-
-  @task
-  *loadFirstAgendaitem() {
-    if (this.args.currentAgenda) {
-      // sorting on show-as-remark prevents defaulting to an announcement when there are notas
-      return yield this.store.queryOne('agendaitem', {
-        'filter[agenda][:id:]': this.args.currentAgenda.id,
-        sort: 'show-as-remark,number',
-      });
-    }
-    return null;
+  get firstAgendaitem() {
```

Comment:
Some explanation on this change.

When deleting the last agendaitem on an agenda, the "Details" tab is still there. But clicking it breaks the app, because it's trying to navigate to an agendaitem that no longer exists. I couldn't find a way to force the `AgendaTabs` component to refire this task, so instead I passed along the agendaitems as part of the route's model to this component. That means that they get fetched from the backend by the `model()` hook of `agenda`'s route and passed down the component chain.